libs/vector_storage/vector_provider/mongodb.py

```diff
@@ -0,0 +1,107 @@
+from abc import ABC
+from typing import List, Optional
+
+from pymongo import MongoClient
```

Comment:
I think we should have schema of mongodb , we should have index of Metadata fields